metadata.json

```diff
@@ -48,16 +47,13 @@
     {
       "operatingsystem": "Fedora",
       "operatingsystemrelease": [
-        "30",
-        "31",
-        "32",
-        "33"
+        "36",
```

Comment:
Fedora 36 is the version before 40 that had its own releases of puppet, because of that I added it to here, but the tests dont run because of it being EOL. I guess I can remove it?

Reply:
Yes, we should remove EOL Fedora 36, in a separate pull request so that the changelog generator picks it up. 👍